docs/02-pages/01-getting-started/11-deploying.mdx

```diff
@@ -0,0 +1,6 @@
+---
+title: How to deploy your Next.js application
```

Comment:
Do we want this here?

Reply:
Since we're moving the page to Self-hosting, we need a /pages doc for just deploying. We can keep it under BYA if you think it's best. I've been trying to mirror the app docs.